### PR TITLE
Update permissions-consent-overview.md

### PR DESCRIPTION
**Pull request by jsoveral**
Adding a note on custom scopes for admin consent flow.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@omondiatieno 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity-platform/permissions-consent-overview.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ Administrators can grant consent for themselves or for the entire organization.
 
 Authentication requests are prompted for admin consent if consent wasn't granted and if one of those high-privilege permissions is requested.
 
+Custom application scopes are not considered high-privilege for administrator consent authentication request purposes.
```
Comment (omondiatieno) [inline docs/identity-platform/permissions-consent-overview.md:84]: I'm not quite sure I understand what you mean by custom application scopes. Do you mean scopes that aren't part of Microsoft Graph scopes?

Comment (omondiatieno) [inline docs/identity-platform/permissions-consent-overview.md:84]:
```suggestion
Permission requests that contain custom application scopes aren't considered high-privilege and thus they don't require admin consent.
```

Comment (omondiatieno) [reply docs/identity-platform/permissions-consent-overview.md:84]: It this statement still inline with the intended meaning? Or can you adjust it to make it even clearer?

**Review by omondiatieno — APPROVED** [review]

**Pull request merged**